<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Azael Avalos &lt;coproscefalo@gmail.com&gt;</author>
 <class>SleepUtils</class>
 <widget class="QWidget" name="SleepUtils">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sleep Utilities</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" alignment="Qt::AlignLeft">
    <widget class="QCheckBox" name="sleep_music_checkbox">
     <property name="toolTip">
      <string>Toggles the Sleep and Music feature.  The laptop speakers remain powered and the line-in jack is used to connect an external device to use the laptop speakers when the computer is asleep or turned off.</string>
     </property>
     <property name="whatsThis">
      <string>Toggles the Sleep and Music feature.  The laptop speakers remain powered and the line-in jack is used to connect an external device to use the laptop speakers when the computer is asleep or turned off.</string>
     </property>
     <property name="text">
      <string>Sleep and Music</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="sleep_charge_label">
     <property name="text">
      <string>USB Sleep and Charge:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="sleep_charge_combobox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select the USB Sleep and Charge mode.  The computer USB ports will remain powered when the computer is asleep or turned off.</string>
     </property>
     <property name="whatsThis">
      <string>Select the USB Sleep and Charge mode.  The computer USB ports will remain powered when the computer is asleep or turned off.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="toolTip">
      <string>Allows the sleep functions to operate when the computer is under battery power.</string>
     </property>
     <property name="whatsThis">
      <string>Allows the sleep functions to operate when the computer is under battery power.</string>
     </property>
     <property name="title">
      <string>Sleep Utilities under battery</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="battery_level_label">
        <property name="text">
         <string>Battery Level:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="battery_level">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The current battery level at which the sleep functions will be disabled.</string>
        </property>
        <property name="whatsThis">
         <string>The current battery level at which the sleep functions will be disabled.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="battery_level_slider">
        <property name="toolTip">
         <string>Sets the battery level at which the sleep functions will be disabled.</string>
        </property>
        <property name="whatsThis">
         <string>Sets the battery level at which the sleep functions will be disabled.</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
